contracts/crowdsale/price/CapStagedCrowdsale.sol

```diff
@@ -0,0 +1,70 @@
+pragma solidity ^ 0.4.24;
+
+import "../Crowdsale.sol";
+import "../../math/SafeMath.sol";
+import "../../ownership/Ownable.sol";
+
+/**
+ * @title CapStagedCrowdsale
+ * @dev Extension of Crowdsale contract that changes the price of tokens regarding of raised ETH. Each stage of crowdsale is defined in StageLimit (cap limit for the stage) and stageRate (stage rate)
+ * Stages must be inserted from first to last with increasing stageLimit.
+ */
+contract CapStagedCrowdsale is Crowdsale, Ownable {
+  using SafeMath for uint256;
+  using SafeMath for uint;
+
+  struct Stage {
+    uint256 stageLimit;
+    uint256 stageRate;
+  }
+
+
+
+  Stage[] public stages;
+
+  constructor
+  (
+    uint256 _rate,
+    address _wallet,
+    ERC20 _token
+  )
+  Crowdsale(_rate, _wallet, _token) 
+  public
+  {
+  }
+
+  /**
+  * @dev Function for adding stages. Stages must be inserted from first to last
+  * @param stLimit - Stage limit in wei, stRate - limit in weiRaise for the stage
+  */
+  function addStage(uint256 stLimit, uint256 stRate) public onlyOwner {
```

Comment:
let's use `_` prefix for arguments here

the function visibility should come at the end like `onlyOwner public`